### DISCUSSION TITLE
React Native: Async Storage depreciado?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1588] [Perguntas e Respostas]
Galera, minha app começou a quebrar falando que o Async Storage está defasado... eu realmente uso ele em minha aplicação, mas se ele está se tronando defasado, o que usar? E como usar?

Minha próxima aplicação é um controle de fluxo de caixa, onde pretendo gravar uma carga de muitos registros no celular do usuário, o que indicam? Usar async storage? Consigo usar Sql lite no react-native? (SQLite) É a melhor opção neste caso?

Muito obrigado, abraços!

**Comment by davipm** [comment]
Isso, foi depreciado, na documentação do RN ele recomenda essa [lib](https://github.com/react-native-community/async-storage) no lugar.

**Comment by kivervinicius** [comment]
@wilsonneto-dev olha o realm
